docs/Documentation/Conditions-List.md

```diff
@@ -210,6 +210,20 @@ This condition requires the player to be _below_ specific Y height. The required
     ```YAML
     height 16
     ```
+    
+## In Conversation: `inconversation`
+
+This condition checks, if the player is in a conversation.
+
+| Parameter      | Syntax              | Default Value            | Explanation                                                                                        |
+|----------------|---------------------|--------------------------|----------------------------------------------------------------------------------------------------|
+| _conversation_ | `conversation:name` | :octicons-x-circle-16:   | Optional name of the conversation. If specified, it will check only for the specific conversation. |
```

Comment:
```suggestion
| _conversation_ | `conversation:name` | :octicons-x-circle-16:   | Optional name of the conversation. If specified, it will only check for the conversation with this name. |
```